dist/index.html

```diff
@@ -388,8 +424,8 @@ <h2 class="branding">Blog</h2>
 					<div class="col-md-6">
 						<h2 class="branding">Recommended Titles</h2>
 						<div class="row">
-							<div class="col-md-4"><img class="book-cover" src="/dist/images/grumpy-cat.jpg" width="100%" height="200"></div>
-							<div class="col-md-8">
+							<div class="col-xs-4 book-cover"><img src="/dist/images/grumpy-cat.jpg" width="100%" height="200"></div>
```

Comment:
not going to prevent approval, but we should talk about class naming conventions. (I think this is ok either way)